packages/build-tooling/src/build-options.ts

```diff
@@ -0,0 +1,144 @@
+import typescript from '@rollup/plugin-typescript'
+import { readFileSync } from 'fs'
+import type { InputPluginOption, Plugin, RollupOptions } from 'rollup'
+import type { BuildOptions, LibraryOptions } from 'vite'
+
+type StrictPluginOptions = RollupOptions & { plugins?: InputPluginOption[] }
+
+/**
+ * Generate rollup options for a Scalar package build
+ * DO NOT USE FOR CLIENT BUNDLING
```

Comment:
Might be just me, but I don’t know what “client bundling” is. Can you help me to understand it? Or maybe we could use something like:

> Generate Rollup options to build packages
>
> Warning: This works for packages only, DO NOT use this to bundle something that is directly served to the user.

And the comment for `createViteBuildOptions` could be the same, as it’s for the same use case, but for Vite, right?